backend/loi-service/src/main/java/org/egov/works/util/MDMSUtils.java

```diff
@@ -1,30 +1,34 @@
 package org.egov.works.util;
 
+import lombok.extern.slf4j.Slf4j;
 import org.egov.common.contract.request.RequestInfo;
-import org.egov.mdms.model.MasterDetail;
 import org.egov.mdms.model.MdmsCriteria;
 import org.egov.mdms.model.MdmsCriteriaReq;
 import org.egov.mdms.model.ModuleDetail;
 import org.egov.works.config.LOIConfiguration;
 import org.egov.works.repository.ServiceRequestRepository;
-import org.egov.works.web.models.LetterOfIndent;
+import org.egov.works.web.models.LetterOfIndentRequest;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.LinkedList;
 import java.util.List;
 
-import static org.egov.works.util.LOIConstants.MDMS_MASTER;
-import static org.egov.works.util.LOIConstants.MDMS_MODULE_NAME;
-
 @Component
+@Slf4j
 public class MDMSUtils {
 
-    private final LOIConfiguration config;
+    private LOIConfiguration config;
+
+    private ServiceRequestRepository serviceRequestRepository;
+
+    public static final String PLACEHOLDER_CODE = "{code}";
+    public final String filterWorksModuleCode = "$.[?(@.active==true && @.code=='{code}')]";
```

Comment:
Which master is this for? Mention the master name in the variable as you have written below for `...subScheme...` and `...subType...`.